data/SugarBean.php

```diff
@@ -2397,6 +2397,10 @@ public function save($check_notify = false)
             $this->custom_fields->save($isUpdate);
         }
 
+        if(empty($this->parent_id)) {
```

Comment:
Hi @cameronblaikie @jack7anderson7, I don't think this is the correct place to do this fix. Not all sugar beans have a parent_type and parent_id. Plus, I think that the flex related fields name can differ